src/app/frontend/replicasetdetail/updatereplicas_controller.js

```diff
@@ -0,0 +1,77 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Controller for the update replica set dialog.
+ *
+ * @final
+ */
+export default class UpdateReplicasDialogController {
+  /**
+   * @param {!md.$dialog} $mdDialog
+   * @param {!angular.$log} $log
+   * @param {!function(number, function(!backendApi.ReplicaSetDetail),
+   * function(!angular.$http.Response)=)} updateReplicasFn
+   * @ngInject
+   */
+  constructor($mdDialog, $log, updateReplicasFn) {
+    /** @export {number} */
+    this.replicas = 1;
```

Comment:
Ok, I think that there should be some description to say that f.e. `5` that is displayed there, is actually current number of pods, because without any information, displaying some number (even current) doesn't mean anything.